pybikes/whitebikes.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Eduardo Mucelli Rezende Oliveira <edumucelli@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class WhiteBikes(BikeShareSystem):
```

Comment:
Can we rename the system to `OpenSourceBikeShare`, since there are other systems that have this same schema. More info at http://opensourcebikeshare.com/